spec/lib/efile/az/az140_calculator_spec.rb

```diff
@@ -647,4 +647,19 @@
       expect(instance.refund_or_owed_amount).to eq(20)
     end
   end
+
+  context "with cents values" do
+    before do
+      allow_any_instance_of(Efile::Az::Az321Calculator).to receive(:calculate_line_13).and_return 30
```

Comment:
i _believe_ this is for line 5c. why are you testing this here when you altered the calculation for line 5?

Reply:
good eye, 5c is unrelated, removed from this spec 👍